packages/dx-react-scheduler/docs/reference/week-view.md

```diff
@@ -143,6 +153,7 @@ Field | Type | Description
 ------|------|------------
 startDate? | Date | Specifies the cell's start time.
 endDate? | Date | Specifies the cell's end time.
+groupOrientation? | [GroupOrientation](./grouping-state.md#grouporientation) | Scheduler's grouping orientation: either 'Vertical' or 'Horizontal'.
```

Comment:
Very strange see it here)

Reply:
It's necessary to use it along with `endOfGroup` prop to render borders correctly